tests/admin_views/tests.py

```diff
@@ -725,6 +725,11 @@ def test_disallowed_to_field(self):
         response = self.client.get(reverse('admin:admin_views_referencedbyinline_changelist'), {TO_FIELD_VAR: 'name'})
         self.assertEqual(response.status_code, 200)
 
+        # #25622 - Specifying a field of model refered by generic relation shouldn't raise an exception.
```

Comment:
The nuance between both `400` is that the exception raised in this case used to result in a `500` when it should have been a `400`. Maybe I should simply use the word "crash" instead of "exception"?